ci/jenkins/jobs/macros.yaml

```diff
@@ -99,6 +100,7 @@
     builders:
       - shell: |-
           #!/bin/bash
+          set -e
```

Comment:
Maybe also add

```
set -x
```

`set -x` makes it prints more debug info. Jenkins sets this by default, too.

Reply:
Updated.